www/docs/server/adapter/standalone.md

```diff
@@ -0,0 +1,151 @@
+---
+id: standalone
+title: Standalone Usage
+sidebar_label: Standalone
+slug: /standalone
+---
+
+## Example app
+
+<table>
+  <thead>
+    <tr>
+      <th>Description</th>
+      <th>URL</th>
+      <th>Links</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>Standalone tRPC Server with Node.js</td>
+      <td><em>n/a</em></td>
+      <td>
+        <ul>
+          <li><a href="https://githubbox.com/trpc/trpc/blob/main/examples/minimal">CodeSandbox</a></li>
```

Comment:
This githubbox link (which I've invented) is not working, does it need configuring by someone in the core team?

Reply:
https://stackblitz.com/github/trpc/trpc/blob/main/examples/minimal

would work although stackblitz is on Node 16 so it complains about missing fetch implementation :/

Reply:
At least that opens the code and makes it browsable rather than fully broken, I've switched out the URL and also added the minimal-react example as that one covers the cors use-case :)